be/src/common/config.h

```diff
@@ -536,6 +536,8 @@ namespace config {
     // Whether to continue to start be when load tablet from header failed.
     CONF_Bool(ignore_load_tablet_failure, "false");
 
+    // Directory where BE's website files placed.
+    CONF_String(www_path, "${DORIS_HOME}/www");
```

Comment:
Is it necessary to be a config? I think it can be hard coded?

Reply:
Seems not necessary, I'll remove it.